covalent/_file_transfer/strategies/s3_strategy.py

```diff
@@ -0,0 +1,77 @@
+import os
+import urllib.request
+from pathlib import Path
+from subprocess import PIPE, CalledProcessError, Popen
+from typing import Optional
+
+from furl import furl
+
+from covalent._file_transfer import File
+from covalent._file_transfer.strategies.transfer_strategy_base import FileTransferStrategy
+
+
+class S3(FileTransferStrategy):
+
+    """
+    Implements Base FileTransferStrategy class to use HTTP to download files from public URLs.
+    """
+
+    def __init__(
+        self,
+        aws_access_key_id: str = os.getenv("AWS_ACCESS_KEY_ID") or None,
```

Comment:
rather than asking for plaintext passwords, users should specify the credentials file.  remove the first three options here and add an option to pass the credentials file and a profile name.  see the fargate executor for an example.

Reply:
Changed to input credential file.